app/scripts/views/confirm.js

```diff
@@ -41,14 +44,23 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
     },
 
     context: function () {
-      return {
+      var context = {
         email: this.getAccount().get('email')
       };
+
+      context.verificationExperiment = this._verificationExperiment.getContext();
+
+      if (this._verificationExperiment.is('openGmail')) {
```

Comment:
Yeah we fixed that. Even if it did it would not clutter the logs, context does not log anything